module/Motion/EdgeChanging.h

```diff
@@ -0,0 +1,36 @@
+/**
+ * @file   EdgeChanging.h
+ * @brief  エッジ切り替え
+ * @author takahashitom
+ */
+
+#ifndef EDGE_CHANGING_H
+#define EDGE_CHANGING_H
+
+#include "Motion.h"
+
+class EdgeChanging : public Motion {
+ public:
+  /**
+   * コンストラクタ
+   * @param _isLeftEdge エッジの左右判定(true:左エッジ, false:右エッジ)
+   * @param _nextEdge 切り替え後のエッジ(true:左エッジ，false:右エッジ)
```

Comment:
```suggestion
   * @param _isLeftEdge エッジの左右判定(true:左エッジ, false:右エッジ)
   * @param _nextEdge 切り替え後のエッジ(true:左エッジ, false:右エッジ)
```

nextEdge の方が全角になってるのかな？

Reply:
レビューありがとうございます。
修正しました。
be577e8a9b7c0fcb132379bf1480999da0cab9a3